build.gradle.kts

```diff
@@ -38,3 +38,17 @@ subprojects {
         }
     }
 }
+
+tasks.register<Exec>(name = "releaseArtMaker") {
```

Comment:
I have seen a problem, the same one you mentioned to Caleb here regarding the input from the gradle task. I have it too. Did not manage to fully get it working. You can delete this task. We can figure more out later. Delete the task but make sure you update the `RELEASE.md` file with steps to trigger this pipeline .